RecoLocalTracker/SiStripClusterizer/plugins/clusterGPU.cuh

```diff
@@ -0,0 +1,31 @@
+#ifndef _CLUSTER_GPU_KERNEL_
+#define _CLUSTER_GPU_KERNEL_
+
+#include "CUDADataFormats/SiStripCluster/interface/GPUtypes.h"
+#include "CUDADataFormats/SiStripCluster/interface/SiStripClustersCUDA.h"
+
+#include <cstdint>
+
+namespace stripgpu {
+  class StripDataGPU;
+}
+struct ChanLocStruct;
+
+static constexpr auto MAX_SEEDSTRIPS = 200000;
+static constexpr uint32_t kClusterMaxStrips = SiStripClustersCUDA::kClusterMaxStrips;
+
+struct sst_data_t {
```

Comment:
may I ask for a more meaningful name ?

Reply:
SST is a standard abbreviation of Silicon Strip Tracker. This looks OK to me.

Reply:
CMS policy is classes and structs should begin with capital letters and use camel case.